lib/wanda_web/views/executions_view.ex

```diff
@@ -0,0 +1,7 @@
+defmodule WandaWeb.ExecutionsView do
+  use WandaWeb, :view
+
+  def render("list_checks_executions.json", %{data: data, total_count: total_count}) do
```

Comment:
I don't see the need to bringing the `total_count` here, we could simply use `length`. I have seen it widely this way. And we need to use `executions` in the 2nd param to follow the convention and make phoenix work properly
```
def render("list_checks_executions.json", %{executions: executions}) do
    %{
      data: render_many(executions, ExecutionView, "execution.json"),
      total_count: length(executions)
    }
  end
```

Reply:
We need the total count because if we paginate, say, 100 entries, with 10 items per page, the total count would still be a hundred while the length of the results we're rendering is 10

Reply:
Anyway I think you're right about the `render_many`, I'll try that :+1: